lib/services/lambda/index.js

```diff
@@ -99,6 +101,9 @@ function getPolicyStatementsForLambdaRole(serviceContext, dependenciesDeployCont
     return deployPhaseCommon.getAllPolicyStatementsForServiceRole(ownPolicyStatements, dependenciesDeployContexts);
 }
 
+function constructRoleName(ownServiceContext) {
```

Comment:
I think you can get rid of this function and use deployPhaseCommon.getResourceName instead. That should be the same as this function, and everyone else uses it, so if we ever change those names it'll be nice to only change that in one place.